other/cross-chain/README.md

```diff
@@ -0,0 +1,31 @@
+---
+description: Use Angle on different chains and layers
+---
+
+# ⛓ Angle cross-chain infrastructure
+
+## Angle on different networks
+
+Angle Protocol is composed of two set of smart contracts: the Core module, and the Borrowing module. 
+
+While the Core module is only deployed on Ethereum mainnet, the Borrowing module is also deployed on other chains and layer 2s. This allows for native minting of agEUR on these other networks. 
+
+{% page-ref page="borrowing-module-cross.md" %}
+
+
+## Bridging between different networks
+
+Bridges are what allow agEUR and the ANGLE token to be available on different chains. At the top level, the bridges for agEUR (and ANGLE) all do the same thing: they lock agEUR (or ANGLE) in a smart contract on the origin network, and mint new tokens on the destination network. When bridging back agEUR (or ANGLE) from the destination chain, tokens are burnt there and released from the smart contract on the origin chain.
+
+AgEUR can be bridged between different networks using our new LayerZero infrastructure. It was previously relying on Multichain, which is still used for ANGLE. 
+
+{% page-ref page="layer-zero-setup.md" %}
+
+{% page-ref page="multichain-setup.md" %}
```

Comment:
Like we need this page on top that says for each chain which bridge solutions are supported for agEUR and ANGLE